<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateApplication</class>
 <widget class="QWidget" name="CreateApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>668</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>AID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="aid">
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
       <property name="placeholderText">
        <string>FFFFFF</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Keys</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="numKeys">
       <property name="maximum">
        <number>14</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Key Change Rights</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="keyChangeRightsNeedsMaster">
        <property name="text">
         <string>Need Master Key</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="keyChangeRightsNeedSpecific">
          <property name="text">
           <string>Need specific key</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="specificKey">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>14</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="keyChangeRightsNeedKey">
        <property name="text">
         <string>Need key to be changed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="keyChangeRightsCanNotChange">
        <property name="text">
         <string>Keys can not be changed (except master key)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Settings Change Rights</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="settingsRightsNotChange">
        <property name="text">
         <string>Settings can not be changed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="settingsRightsMasterChange">
        <property name="text">
         <string>Can be changed with master key</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>File Alter Rights</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="fileAlterRightsNeedMaster">
        <property name="text">
         <string>Need master key</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fileAlterRightsNoMaster">
        <property name="text">
         <string>Allowed without master key</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Directory Access Rights</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QRadioButton" name="directoryAccessRightsNeedMaster">
        <property name="text">
         <string>Need master key</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="directoryAccessRightsFreeListening">
        <property name="text">
         <string>Free file listing/accessing</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Master Key Rights</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QRadioButton" name="masterKeyRightsMasterNotChange">
        <property name="text">
         <string>Master key not changeable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="masterKeyRightsNeedMaster">
        <property name="text">
         <string>Need master key to change master key</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateApplication</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>646</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
